packages/at_utils/lib/src/logging/atsignlogger.dart

```diff
@@ -7,11 +7,12 @@ import 'package:logging/logging.dart' as logging;
 /// Class for AtSignLogger Implementation
 class AtSignLogger {
   late logging.Logger logger;
-  static String _root_level = 'info';
-  bool _hierarchicalLoggingEnabled = false;
   String? _level;
 
-  static var loggingHandler = ConsoleLoggingHandler();
+  static String _root_level = 'info';
+  static LoggingType loggingType = LoggingType.console;
+  static var loggingHandler = _getLoggingHandler(loggingType);
```

Comment:
This will be set the first time an AtSignLogger is constructed, and is never subsequently be changed. This behaviour needs to be documented in the code

However, ideally I would like the logging handler to be selectable at the time each individual AtSignLogger is constructed, in addition to there being a default which is settable for all 

Done this way, loggingHandler wouldn't be a public static; instead you'll have two private static vars, a _consoleLoggingHandler and a _stdErrLoggingHandler. You would also rename loggingType to `defaultLoggingType`, and you would add loggingType as an optional parameter to the AtSignLogger constructor

Reply:
The changes are implemented and pushed. Please review